app/src/main/java/it/quip/android/adapter/NotificationAdapter.java

```diff
@@ -0,0 +1,116 @@
+package it.quip.android.adapter;
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.text.Html;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.squareup.picasso.Picasso;
+import java.util.List;
+
+import it.quip.android.R;
+import it.quip.android.model.Notification;
+
+/**
+ * Created by danbuscaglia on 10/18/15.
+ */
+public class NotificationAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+
+    private final int UNREAD = 0, READ = 1;  // State
```

Comment:
Ah scratch that, looks like you're using it for view type. This is good then. :+1: